rust/azure_iot_operations_services/src/azure_device_registry/models/asset.rs

```diff
@@ -193,9 +193,9 @@ pub struct DiscoveredDatasetDataPoint {
 /// Represents the destination for a dataset.
 #[derive(Clone, Debug)]
 pub struct DatasetDestination {
-    /// The configuration for the destination
+    /// The destination configuration.
```

Comment:
could this be more descriptive ?

Reply:
This is directly what's from the DTDL - I don't want to differ from it too much unless we already had something way better or there's something that isn't relevant for our audience